user/common-build-problems.md

```diff
@@ -462,37 +462,56 @@ which Docker image you are using on Travis CI.
 
 ### Running a Container Based Docker Image Locally
 
-* Download and install Docker:
+1. Download and install Docker:
 
    - [Windows](https://docs.docker.com/docker-for-windows/)
    - [OS X](https://docs.docker.com/docker-for-mac/)
    - [Ubuntu Linux](https://docs.docker.com/engine/installation/linux/ubuntulinux/)
 
-* For Ubuntu 12.04 (precise), select an image from
-   [Quay.io](https://quay.io/organization/travisci) named `travis-{lang}` where
-`{lang}` is the language you need.  If you're not using a language-specific
-image, pick `travis-ruby`.  For Ubuntu 14.04 (trusty), select an image from
-[Docker Hub](https://hub.docker.com/r/travisci/) named either `ci-amethyst` or
-`ci-garnet` (which differ slightly depending on language desired).  In order for
-system services to run correctly, the container must be run with `/sbin/init` as
-PID 1:
+1. Choose a Docker image
+  * For Ubuntu 12.04 (precise), select an image from
+    [Quay.io](https://quay.io/organization/travisci) named `travis-{lang}` where
+    `{lang}` is the language you need.  If you're not using a language-specific
+    image, pick `travis-ruby`.
+  * For Ubuntu 14.04 (trusty), select an image from
+    [Docker Hub](https://hub.docker.com/r/travisci/) named either `ci-amethyst` or
+    `ci-garnet` (which differ slightly depending on language desired).
```

Comment:
Should we include ci-sugilite? Maybe we should also include a mapping of image vs supported language, WDYT?